tests/custom/test_sdk.py

```diff
@@ -69,9 +69,11 @@ def test_entities(setup_connection):
     # test the GET variant of the search endpoint
     entity_search_get_results = client.search.search_entity_get(q=random_string)
     assert len(entity_search_results.data) == len(entity_search_get_results.data)
-    assert entity_search_results.size.count == entity_search_get_results.size.count
     assert entity_search_results.size.qualifier == entity_search_get_results.size.qualifier
 
+    #the difference in counts should be less than 30%
+    assert abs(entity_search_results.size.count - entity_search_get_results.size.count)/entity_search_results.size.count < .3
```

Comment:
yep, only risk here is a divby0 if there are no results...but I think that's ok since that should probably fail the test also

Reply:
Reasonable concern, but a new search is performed if the search doesn't return results. https://github.com/sayari-analytics/sayari-python/blob/6704fabf217a602645571df4970c3abe2ba5b7e7/tests/custom/test_sdk.py#L64